packages/snap/src/rpc/substrate/sendUnit.ts

```diff
@@ -0,0 +1,19 @@
+import ApiPromise from "@polkadot/api/promise";
+import {Wallet} from "../../interfaces";
+import { Keyring } from "@polkadot/api";
+import { Hash } from '@polkadot/types/interfaces/runtime';
+
+export async function sendUnit(wallet: Wallet, api: ApiPromise, amount: string|number, to: string): Promise<Hash> {
+  const keyring = new Keyring();
+  const keypair = keyring.addFromJson(wallet.getPluginState().polkadot.account.keyring);
+  const { nonce } = await api.query.system.account(keypair.address);
+  const transfer = api.tx.balances.transfer(to, amount);
+  return new Promise((resolve) => {
+    transfer.signAndSend(keypair, {nonce}, ({ events = [], status }) => {
+      if (status.isInBlock) {
```

Comment:
Maybe it's better to return hash, and expose exevnt listener for inBlock and finalized but we can do that in separate pr

Reply:
I will refactor `sendUnit` function to return a hash for now and create issues for creating event listeners for inBlock and finalized.